vendor/hound/config/style_guides/ruby.yml

```diff
@@ -60,7 +60,7 @@ Naming/PredicateName:
   - is_
```

Comment:
Is this file necessary for hound to work?
Actually I had removed the `vendor/hound/config/style_guides/ruby.yml` when i started making changes.

Reply:
I'm not sure, but I can try removing it.

Reply:
Merged and removed.